### PR TITLE
Fireball aiming

### PR DESCRIPTION
**Pull request by ghost**
:cl:  
rscadd: The fireball spell can now be properly aimed
fix: since i'm lazy, i fixed things up to sorta compile on 512
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
I made this by cherry-picking commit ```e2cfcb0bd191af1c205205a955ae1f8f58eda416``` from /tg/station and then fixing the conflicts and errors

**Comment by tsit9** [comment]
Does this work while stunned?

**Comment by ghost** [comment]
i think so, but if you're stunned, you'll probably hit the person that is running towards your face, which will likely result with you in the blast zone, too.

**Comment by Partheo** [comment]
Did you test it

**Comment by Time-Green** [comment]
We already had an aimable fireball spell

**Comment by ghost** [comment]
@Partheo yes

**Comment by ghost** [comment]
At least this fireball can't be dropped when you're stunned, so this is better.

File: code/game/gamemodes/shadowling/shadowling_abilities.dm [hunk 1]
```diff
@@ -895,7 +895,6 @@
 	human_req = 1
 	clothes_req = 0
 	action_icon_state = "darksight"
-	var/active = 0
```
Comment (Kmc2000) [inline code/game/gamemodes/shadowling/shadowling_abilities.dm:898]: Intentional?

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 2]
```diff
@@ -499,7 +499,7 @@
 /obj/item/weapon/storage/backpack/dufflebag/syndie/firestarter/New()
 	..()
 	new /obj/item/clothing/under/syndicate/soviet(src)
-	new /obj/item/weapon/watertank/operator(src)
+	new /obj/item/weapon/watertank/op(src)
```
Comment (Kmc2000) [inline code/game/objects/items/weapons/storage/backpack.dm:502]: Why are you changing things like this in a fireball PR?

**Comment by ghost** [comment]
this is /tg/ fireball code, so this is how they work: 

https://gfycat.com/MixedSimilarErin
abc

**Review by Kmc2000 — APPROVED** [review]
Looks pretty good, playtesting / footage would be nice.

File: code/game/objects/items/weapons/storage/backpack.dm (hunk 2 above)
Comment (ghost) [reply code/game/objects/items/weapons/storage/backpack.dm:502]: i was too lazy to download 511

**Pull request closed without merging**